packages/contracts-sdk/src/OraiswapV3.types.ts

```
@@ -180,6 +181,8 @@ export type QueryMsg = {
   admin: {};
 } | {
   protocol_fee: {};
+} | {
+  incentives_fund_manager: {};
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)